app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/LoginScreen.kt

```diff
@@ -56,10 +63,38 @@ fun LoginScreen(
     LoginAppLogo(modifier = Modifier.weight(1f))
     WelcomeTitle(modifier = Modifier.weight(4f))
     SignInButton(modifier = Modifier.weight(1f)) {
-      val intent =
-          AuthorizationClient.createLoginActivityIntent(
-              context.getActivity(), viewModel.getAuthorizationRequest())
-      launcher.launch(intent)
+      // val intent =
```

Comment:
is it normal that this is commented?

Reply:
This is just for testing purposes, I will get rid of it once the or is ready and I did the testing